From e74f6eee3152a9b505d7181290fe8e2661f6f5b6 Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 22 Jul 2020 09:43:29 +0700
Subject: [PATCH 279/476] ARM: dts: r8a7744: Add 3DX graphics (PowerVR
 SGX544MP2) node

Add definition of 3DX graphics module to the R8A7744 (RZ/G1N) SoC dtsi.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 282466e..719b4da 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -285,6 +285,17 @@
 			reg = <0 0xe6060000 0 0x250>;
 		};
 
+		sgx@fd800000 {
+			compatible = "renesas,sgx";
+			reg = <0 0xfd800000 0 0x10000>;
+			interrupts = <0 119 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-names = "irq_sgx";
+			clocks = <&mstp1_clks R8A7744_CLK_3DG>,
+				 <&mstp8_clks R8A7744_CLK_IPMMU_SGX>;
+			clock-names = "sgx_clk", "ipmmu_clk";
+			power-domains = <&sysc R8A7744_PD_SGX>;
+		};
+
 		tpu: pwm@e60f0000 {
 			compatible = "renesas,tpu-r8a7744", "renesas,tpu";
 			reg = <0 0xe60f0000 0 0x148>;
-- 
2.7.4

